multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -32,13 +32,19 @@
 except ImportError:
     mmdet = None
 
+try:
+    import mmocr
+except ImportError:
+    mmdet = None
```

Comment:
Should it be `mmocr = None`?

Reply:
Yes, it is my mistake. I will modify it.